client/blocks/post-item/index.jsx

```diff
@@ -125,6 +126,17 @@ class PostItem extends React.Component {
 		);
 	}
 
+	renderEllipsisMenu() {
+		const { globalId, isTypeWpBlock, multiSelectEnabled } = this.props;
+		if ( multiSelectEnabled ) {
+			return null;
+		}
+		if ( isTypeWpBlock ) {
```

Comment:
Do we need a whole new `WpBlockActionsEllipsisMenu` component? Could we just use the `children` property of `PostActionsEllipsisMenu` instead to pass in `PostActionsEllipsisMenuEdit` and `PostActionsEllipsisMenuTrash` conditionally on `isTypeWpBlock`?

Reply:
Both the alternatives look a bit off to me:
- Passing the menu items as children in `PostItem` would mean importing them into an already crowded component.
- Passing them in `PostActionsEllipsisMenu` would create a specific exception.

Though, I don't have a strong opinion on this, and I definitely agree that the file name and location are weird as hell. 🤔